assignment1/fc_net.py

```diff
@@ -99,6 +99,46 @@ def update_parameters(self, parameters, grads, learning_rate):
 
         return parameters
 
+    def L_layer_model(self, X, Y, layers_dims, learning_rate, num_iterations, batch_size):
+        """
+
+        :param X: the input data, a numpy array of shape (height*width , number_of_examples)
+        :param Y: the “real” labels of the data, a vector of shape (num_of_classes, number of examples)
+        :param layers_dims: a list containing the dimensions of each layer, including the input
+        :param learning_rate: the learning rate
+        :param num_iterations: number of iterations
+        :param batch_size: the number of examples in a single training batch.
+        :return: (parameters, costs) - the parameters learnt by the system during the training (the same parameters
+                                        that were updated in the update_parameters function) and the values of the cost
+                                        function (calculated by the compute_cost function). One value is to be saved
+                                        after each 100 training iterations (e.g. 3000 iterations -> 30 values)..
+        """
+        # initialization
+        parameters = initialize_parameters(layers_dims)
```

Comment:
we already call this function in __init__
we should think how to organize it

Reply:
yes i agree, need to be structured better